downstream/modules/platform/proc-controller-importing-subscriptions.adoc

```diff
@@ -34,25 +46,26 @@ After you enter your credentials, click btn:[Get Subscriptions].
 Then, it prompts you to select the subscription that you want to run and applies that metadata to {ControllerName}. 
 You can log in over time and retrieve new subscriptions if you have renewed.
 +
-. Click btn:[Next] to proceed to the *Tracking and Insights* page. 
-+
-Tracking and insights collect data to help Red Hat improve the product and deliver a better user experience. 
+. Click btn:[Next] to proceed to the End User Agreement.
+//[ddacosta - removed analytics selection for AAP-30863 and AAP-29909] to proceed to the *Tracking and Insights* page. 
+//+
+//Tracking and insights collect data to help Red Hat improve the product and deliver a better user experience. 
 //For more information about data collection, see link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_administration_guide/index#controller-usability-analytics-data-collection[Usability Analytics and Data Collection] of the _{ControllerAG}_.
-+ 
-This option is checked by default, but you can opt out of any of the following:
+//+ 
+//This option is checked by default, but you can opt out of any of the following:
 
 //* *User analytics*. Collects data from the controller UI.
-* *Insights Analytics*. Provides a high level analysis of your automation with {ControllerName}. 
-It helps you to identify trends and anomalous use of the controller. 
-For opt-in of {Analytics} to be effective, your instance of {ControllerName} must be running on {RHEL}. 
-For more information, see the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_administration_guide/index#ref-controller-automation-analytics[{Analytics}] section. TBD
-+
-[NOTE]
-====
-You can change your analytics data collection preferences at any time.
-====
-+
-. After you have specified your tracking and Insights preferences, click btn:[Next] to proceed to the End User Agreement.
+//* *Insights Analytics*. Provides a high level analysis of your automation with {ControllerName}. 
+//It helps you to identify trends and anomalous use of the controller. 
+//For opt-in of {Analytics} to be effective, your instance of {ControllerName} must be running on {RHEL}. 
+//For more information, see the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/configuring_automation_execution/index#ref-controller-automation-analytics[{Analytics}] section of _{ControllerAG}_.
+//+
+//[NOTE]
+//====
+//You can change your analytics data collection preferences at any time.
+//====
+//+
+//. After you have specified your tracking and Insights preferences, click btn:[Next] 
 . Review and check the *I agree to the End User License Agreement* checkbox and click btn:[Submit].
```

Comment:
According to the rich diff, looks like the numbering restarts with line 69 ("Review and check..."). I think a + in the line above should fix this.

Reply:
It's hard to see but this should be ok because line 69 follows line 49 now with everything in between commented out.